content/en/examples/examples_test.go

```diff
@@ -404,6 +404,10 @@ func TestExampleObjectSchemas(t *testing.T) {
 		"application/zookeeper": {
 			"zookeeper": {&api.Service{}, &api.Service{}, &policy.PodDisruptionBudget{}, &apps.StatefulSet{}},
 		},
+		"configmap": {
+			"configmaps":            {&api.ConfigMap{}},
+			"configmap-multikeys":   {&api.ConfigMap{}, &api.ConfigMap{}},
```

Comment:
This is funny, you have configmaps.yaml listing two `ConfigMap` resources and you are telling the test cases that the `configmap-multikeys` has two `ConfigMap`s in it.

Reply:
yes and i was expecting to work but yet, no dice. Am i missing something @tengqm  ?

Same thing in [here](https://github.com/kubernetes/website/blob/master/content/en/examples/examples_test.go#L395) works

Reply:
note even with `"configmap-multikeys":   {&api.ConfigMap{}},` it fails